spec/licensee/matchers/dice_matcher_spec.rb

```diff
@@ -80,4 +80,39 @@
       end
     end
   end
+
+  context 'confidence similarity match' do
+    module Licensee
+      module ContentHelper
+        alias max_delta_original max_delta
+        alias max_delta calculate_max_delta
+        public :max_delta
+      end
+    end
+
+    Licensee.licenses(hidden: true).each do |license|
+      next if license.pseudo_license?
+      context "the #{license.key} license" do
+        let(:content) { license.content }
+
+        nearest =
+          Licensee
+          .licenses(hidden: true)
+          .reject { |other| other.pseudo_license? || other.key == license.key }
+          .collect { |other| [other, license.similarity(other)] }
+          .max_by { |other_similarity| other_similarity[1] }
+        next if nearest[1] < 50
```

Comment:
What's going on here? I think this logic could use a comment.

Reply:
Adding a comment here (and a few others), but in short we're finding the most similar license to the one we're looking at (we're going to see if it is in the set of similar licenses, not cut off from comparison due to max delta), but not if confidence is set to less than 50 in which case all bets are off, and who would want to set confidence so low anyway (in fact only one pair fails without this).